src/main/java/uk/gov/companieshouse/web/companydocuments/controller/NavigationUrls.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.companieshouse.web.companydocuments.controller;
+
+public interface NavigationUrls {
```

Comment:
Sonar is complaining about this and I agree: you (generally) shouldn't keep constants in an interface. Especially one that you do not implement anywhere.
This should either be an enum or a constants class (with a private constructor).